core/src/main/scala/cats/data/EitherT.scala

```diff
@@ -447,6 +447,12 @@ final case class EitherT[F[_], A, B](value: F[Either[A, B]]) {
         }
     })
 
+  def semiflatTap[C](f: B => F[C])(implicit F: Monad[F]): EitherT[F, A, B] =
+    semiflatMap(b => Monad[F].as(f(b), b))
```

Comment:
This is a nitpick, but for the sake of consistency and to keep things simple, what do you think about using `F.as` here instead of `Monad[F].as`?

Reply:
sure, will do ;)